index.html

```diff
@@ -0,0 +1,666 @@
+            </div>
+            <div class="mobile-nav1">
+                <div class="mobile-nav-search">
+                    <i class="fas fa-search gray"></i>
+                </div>
+                <div class="mobile-nav-more">
+                    <i class="fas fa-bars orange"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <div class="section1">
+                <img src="img/tnw.svg" alt="The Next Web Logo">
+                <div class="showcase-text">
+                    <h6 class="white showcase-header">INSIGHTS</h6>
+                    <h1 class="white showcase-title">Satoshi Nakamoto: ‘Bitcoin crosses $10K for first time in 2 months’
+                    </h1>
+                    <ul class="gray showcase-social fsmall">
+                        <li>by <a class="gray" href="#">Satoshi Nakamoto</a></li>
+                        <li><i class="far fa-clock"></i> 25 minutes ago</li>
+                        <li><i class="fas fa-share-alt"></i> 6</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="section2">
+                <h4 class="white">India is reportedly looking to ban 275 more Chinese apps — including PUBG</h4>
+                <ul class="gray showcase-social fsmall">
+                    <li>by <a class="gray" href="#">Satoshi Nakamoto</a></li>
+                    <li><i class="far fa-clock"></i> 25 minutes ago</li>
+                    <li><i class="fas fa-share-alt"></i> 6</li>
+                </ul>
+            </div>
+            <div class="section3">
+                <h4 class="white">India is reportedly looking to ban 275 more Chinese apps — including PUBG</h4>
+                <ul class="gray showcase-social fsmall">
+                    <li>by <a class="gray" href="#">Satoshi Nakamoto</a></li>
+                    <li><i class="far fa-clock"></i> 25 minutes ago</li>
+                    <li><i class="fas fa-share-alt"></i> 6</li>
+                </ul>
+            </div>
+        </section>
+        <div class="news-pages">
+        <h2 class="orange news-pages-header">Latest News <i class="fas fa-chevron-right"></i></h2>
+        <section class="latest-news">
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
```

Comment:
- [ ] On the mobile screen size, the alignment seems off, and the image size too big.  Try to make it look closer to the original.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88700239-c4b90b80-d0de-11ea-9141-edf9f5a14ed8.png) | ![image](https://user-images.githubusercontent.com/61479648/88700257-cda9dd00-d0de-11ea-933c-d85cd238bebd.png) |